README.md

```diff
@@ -20,14 +20,16 @@ The Docker compose file at `test/docker-compose.yml` can be used to run the requ
 
```

Comment:
Not part of this PR, but I feel like the ReadMe should be more detailed, in following areas:
1. What data does the source connector pull? Currently, it only says data, but we should be more specific.
2. We should also mention what changes are detected.
3. Currently I see no integration tests, even though it was said in the PR they will be added.
4. Explanation of the snaphot mode (how does it work), how does it switch to the CDC mode, and explanation of the CDC mode (how does it work).

Reply:
Readme file updated